src/app/App.js

```diff
@@ -1,31 +1,63 @@
 /** @flow
  * @format */
+
+import '../globals';
```

Comment:
Do we need to import `globals` here too? 🤔

Reply:
Shouldn't be required, removing it 👍